### PR TITLE
Add contribution guidelines

### LINKED ISSUE #75: Provide proper contributor information
Write a large blurb on the documentation describing how from nothing to virtual environment and pre-commit and PR people can contribute.

### PR DESCRIPTION
**Pull request by shtlrs**
Closes #75

This is just a draft as I don't know what exactly is supposed to be detailed into the contribution guideline, and also how detailed it needs to be without boring people ou.

@supakeen This one will need feedback and preferably more explicit details about how we'd want to this/what we want to include, and I'll update it accordingly.

### CODE REVIEW

**Comment by shtlrs** [comment]
@supakeen Do  you think we should include a section where we detail how to choose an issue to work on ?
I don't know how you usually prefer things to get done, meaning whether you prefer that people ask if they can work on an issue or not and have them assigned, or if people can just grab an unassigned issue & go for it.

**Comment by shtlrs** [comment]
@supakeen do you want me to keep this or shall I close it?

**Comment by supakeen** [comment]
@shtlrs Keep it, I'll take a look tomorrow (or today).

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* [Make a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) of the [pinnwand repo](https://github.com/supakeen/pinnwand).
+  * This will have a copy of `pinnwand` as your remote repository on `Github`
+
+* [Clone your forked repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+  * This will allow you to pull/push changes to your remote `pinnwand` fork.
+
+## Setting up the environment
+This can be done in multiple ways. However, `pinnwand` used [PDM](https://pdm-project.org/latest/) to manage dependencies so we'll explain how to setup your environment with it.
+
+* [Install DPM](https://pdm-project.org/latest/#installation)
```
Comment (supakeen) [inline CONTRIBUTING.md:18]: `PDM`

**Review by supakeen — APPROVED** [review]

**Comment by supakeen** [comment]
Actually this is good as-is; we can iterate on it.

**Pull request merged**